localisation/00_wc_random_artifact_names.csv

```diff
@@ -21,60 +37,60 @@ artifact_rare_origin;Fables tell that [GetArtifactRareOrigin].;Des fables disent
+artifact_uncommon_origin_2;[GetGrammaticalNumberThisThese] [This.GetName] [GetGrammaticalNumberWasWere] fished out of a lake;ce [This.GetName] a �t� p�ch� dans un lac;diese [This.GetName] aus einem See gefischt wurde;;;;;;;;;;;x
+artifact_uncommon_origin_3;[GetGrammaticalNumberThisThese] [This.GetName] [GetGrammaticalNumberIsAre] an underwhelming replica of a legendary original;ce [This.GetName] est la r�plique d'un original l�gendaire;diese [This.GetName] ist eine unvollkommene Nachbildung eines legend�ren Originals ist;;;;;;;;;;;x
+artifact_uncommon_origin_4;traces of the original owner's blood can still be seen on [GetGrammaticalNumberThisThese] [This.GetName];des traces du sang des propri�taires originaux peuvent encore �tre observ�es sur ce [This.GetName].;;;;;;;;;;;;x
+artifact_uncommon_origin_5;[GetGrammaticalNumberThisThese] [This.GetName] [GetGrammaticalNumberIsAre] designed after one worn by a famed hero;ce [This.GetName] est con�u d'apr�s celui port� par un h�ros c�l�bre;;;;;;;;;;;;x
+artifact_uncommon_origin_6;[GetGrammaticalNumberThisThese] [This.GetName] [GetGrammaticalNumberWasWere] stolen from a demon;ce [This.GetName] a �t� vol� � un d�mon;;;;;;;;;;;;x
+artifact_uncommon_origin_7;[GetGrammaticalNumberThisThese] [This.GetName] matches the description of one[GetGrammaticalNumberEndingPluralS] worn by a hero in a popular children's lullaby;ce [This.GetName] correspond � la description de celui port� par un h�ros dans une comptine populaire pour enfants;;;;;;;;;;;;x
+artifact_uncommon_origin_8;[GetGrammaticalNumberThisThese] [This.GetName] [GetGrammaticalNumberWasWere] found in the city's library along with other debris gathered in memory of an unknown conflict;ce [This.GetName] a �t� trouv� dans la biblioth�que de la ville avec d'autres d�bris rassembl�s en m�moire d'un conflit inconnu;;;;;;;;;;;;x
+artifact_uncommon_origin_9;[GetGrammaticalNumberThisThese] [This.GetName] [GetGrammaticalNumberHasHave] been tainted by some strange magic;ce [This.GetName] a �t� souill� par une �trange magie;;;;;;;;;;;;x
+artifact_uncommon_origin_10;the appearance of [GetGrammaticalNumberThisThese] [This.GetName] [GetGrammaticalNumberHasHave] been altered by someone to hide [GetGrammaticalNumberItsTheir] identity;l'apparence de ce [This.GetName] a �t� modifi�e par quelqu'un pour cacher son identit�;;;;;;;;;;;;x
+;;;;;;;;;;;;;;x
+artifact_rare_origin_1;[GetGrammaticalNumberThisThese] [This.GetName] once [GetGrammaticalNumberWasWere] a family heirloom of a fallen noble;ce [This.GetName] �tait autrefois l'h�ritage familial d'un noble d�chu;;;;;;;;;;;;x
+artifact_rare_origin_2;[GetGrammaticalNumberThisThese] [This.GetName] supposed to be cursed, but for some reason became enchanted instead;ce [This.GetName] �tait cens� �tre maudit, mais pour une raison quelconque, il est plut�t devenu enchant�;;;;;;;;;;;;x
```

Comment:
Added `[GetGrammaticalNumberWasWere]` behind `[This.GetName]`.
```suggestion
artifact_rare_origin_2;[GetGrammaticalNumberThisThese] [This.GetName] [GetGrammaticalNumberWasWere] supposed to be cursed, but for some reason became enchanted instead;ce [This.GetName] �tait cens� �tre maudit, mais pour une raison quelconque, il est plut�t devenu enchant�;;;;;;;;;;;;x
```

Reply:
Done